labapi/src/main/java/com/microsoft/identity/internal/test/labapi/Configuration.java

```diff
@@ -14,6 +14,7 @@
 
 @javax.annotation.Generated(value = "io.swagger.codegen.v3.generators.java.JavaClientCodegen", date = "2021-06-01T10:19:44.716-07:00[America/Los_Angeles]")public class Configuration {
     private static ApiClient defaultApiClient = new ApiClient();
+    private static ApiClient functionApiClient = new ApiClient("https://thefunctionapi.azurewebsites.net");
 
```

Comment:
IIRC, this file was auto generated by swagger code generator.
If we ever regenerate the code using that tool, how do we ensure that we persist these custom edits.

Reply:
same in other places.